composer.json

```diff
@@ -29,30 +32,30 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0",
+        "ext-json": "*",
+        "api-skeletons/doctrine-orm-hydration-module": "^1.0",
```

Comment:
this should probably be replaced with [doctrine/doctrine-laminas-hydrator](https://github.com/doctrine/doctrine-laminas-hydrator)

Reply:
> this should probably be replaced with [doctrine/doctrine-laminas-hydrator](https://github.com/doctrine/doctrine-laminas-hydrator)

Probably, but it might be not so easy.